ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -290,13 +290,10 @@ objects:
           # This can be used if we just want to check the installer exits 0
           echo "WARNING: No tests were run against the installed cluster"
         }
-
         ${TEST_COMMAND}
 
     # Runs an install
     - name: setup
-      # A midstep till we have the installer work merged, then we
-      # can use the CI artifact
```

Comment:
Looks like this is your comment from #4340.  Can you explain (ideally in the commit message), why you're removing it here?

Reply:
I will edit the commit message